src/components/Menu/Dropdown.tsx

```diff
@@ -0,0 +1,38 @@
+import * as DropdownMenu from '@radix-ui/react-dropdown-menu'
+import { socialLinks } from './utils/socialLinks'
+
+interface DropdownProps {
+  children: React.ReactNode
+}
+
+export function Dropdown({
+  children
+}: DropdownProps) {
+  return (
+    <DropdownMenu.Root>
+      <DropdownMenu.Trigger asChild>
+        {children}
+      </DropdownMenu.Trigger>
+
+      <DropdownMenu.Portal>
+        <DropdownMenu.Content side='right' className="bg-[#2a273f] z-10 flex flex-col gap-2 rounded-lg overflow-auto">
+          {socialLinks.map(socialItem => {
+            return (
+              <DropdownMenu.Item key={socialItem.url} className="outline-none">
+                <a
+                  href={socialItem.url}
+                  target="_blank"
+                  rel="noreferrer"
+                  className="text-[#E0DEF2] flex text-sm px-4 py-2 hover:bg-[#454066]"
```

Comment:
The hover color here is a little brighter than in other places, can we keep the `#2a273f` on hover?

Reply:
#2a273f is the background-color here, what do you suggest ?

Reply:
It's ok like this. Can you please just adjust the arrow color to match the background? 
<img width="35" alt="CleanShot 2023-02-16 at 08 47 26@2x" src="https://user-images.githubusercontent.com/2254731/219356856-9cce2920-f29f-4abe-be66-dbc246277886.png">

Reply:
done